src/sentry/static/sentry/app/views/projectDebugFiles.jsx

```diff
@@ -0,0 +1,252 @@
+import {Box, Flex} from 'grid-emotion';
+import React from 'react';
+import createReactClass from 'create-react-class';
+import {browserHistory} from 'react-router';
+import {omit, isEqual} from 'lodash';
+import qs from 'query-string';
+
+import {Panel, PanelBody, PanelHeader, PanelItem} from 'app/components/panels';
+import {t} from 'app/locale';
+import ApiMixin from 'app/mixins/apiMixin';
+import ActionLink from 'app/components/actions/actionLink';
+import EmptyStateWarning from 'app/components/emptyStateWarning';
+import FileSize from 'app/components/fileSize';
+import InlineSvg from 'app/components/inlineSvg';
+import LoadingError from 'app/components/loadingError';
+import LoadingIndicator from 'app/components/loadingIndicator';
+import OrganizationState from 'app/mixins/organizationState';
+import SettingsPageHeader from 'app/views/settings/components/settingsPageHeader';
+import TextBlock from 'app/views/settings/components/text/textBlock';
+import TimeSince from 'app/components/timeSince';
+import Pagination from 'app/components/pagination';
+import SearchBar from 'app/components/searchBar';
+import LinkWithConfirmation from 'app/components/linkWithConfirmation';
+
+const ProjectDebugSymbols = createReactClass({
+  displayName: 'ProjectDebugSymbols',
+  mixins: [ApiMixin, OrganizationState],
```

Comment:
I thought we were preferring es6 classes and higher order components for setting organization state? cc @billyvg